src/main/scala/org/dbpedia/lookup/server/Result.scala

```diff
@@ -33,7 +33,7 @@ class Result(uri: String,
 
     // make a category label from a category URI
     def categoryLabel(categoryUri: String): String = {
-        WikiUtil.wikiDecode(categoryUri.replace("http://dbpedia.org/resource/Category:", ""))
+        WikiUtil.wikiDecode(categoryUri.replace("http://fr.dbpedia.org/resource/Catégorie:", ""))

```

Comment:
Hi @mullekay .  In order to have a complete i18n support , is a good idea to have it out of the code